### DISCUSSION TITLE
Is the MKL-pardiso solver still available to use in osqp python?

### DISCUSSION THREAD

**Discussion by yeasmin62** [osqp/osqp#678] [Q&A]
I have installed MKL libraries, CMAKE, and trying to connect with them with osqp so that I can use MKL_pardiso solver from python interface but I am seeing that osqp.c and lin_sys.c files are missing in the current directory. Am I missing anything?

**Comment by imciner2** [comment]
The MKL Pardiso solver has been moved to a dedicated backed, which is accessed using the MKL algebra. This backend now also includes a conjugate gradient using MKL as well.

**Reply by yeasmin62** [reply]
thank you very much. MKL now works in my osqp solver.
